source/JsMaterialX/JsMaterialXCore/JsGeom.cpp

```diff
@@ -1,58 +1,69 @@
+#include "../vectorHelper.h"
 #include "../helpers.h"
+
 #include <MaterialXCore/Geom.h>
 
-#include <emscripten.h>
 #include <emscripten/bind.h>
 
 namespace ems = emscripten;
 namespace mx = MaterialX;
 
 #define BIND_GEOMINFO_FUNC_INSTANCE(NAME, T) \
```

Comment:
As a note. "#define" aren't really used for the library and "using" is used instead. I'm not sure if it applies to hear bit a merge head-up.